bundle/edu.gemini.qv.plugin/src/main/scala/edu/gemini/qv/plugin/filter/core/Filter.scala

```diff
@@ -213,10 +214,10 @@ object Filter {
     def label = "RA"
     def lowest = RA.MinValue
     def highest = RA.MaxValue
-    def getter = {o: Obs => o.getRa/15.0 }
+    def getter = (o: Obs, ctx: QvContext) => o.raDeg(ctx) / 15.0
```

Comment:
Yes, well spotted. This is what that provoked all the other QV changes. The method `.raDeg` is available by an implicit conversion in the `edu.gemini.qv.plugin` package object [here](https://github.com/tpolecat/ocs/blob/4b5cff17929aab7255599e5d095cb1570554a974/bundle/edu.gemini.qv.plugin/src/main/scala/edu/gemini/qv/plugin/package.scala#L14-L31).